### PR TITLE
Move cloud to the new registry and add additional fields from ECS

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
## Changes

This PR does two things:
1. move original cloud fields to the registry
2. add additional fields from ECS: 
cloud.service.name
cloud.account.name
cloud.instance.id
cloud.instance.name
cloud.machine.type
cloud.project.id
cloud.project.name
 
## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [x] [CHANGELOG.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CHANGELOG.md) updated for non-trivial changes.
* [ ] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

File: docs/resource/cloud.md [hunk 1]
```diff
@@ -39,50 +39,6 @@ The following well-known definitions MUST be used if you set this attribute and
   `/subscriptions/<SUBSCIPTION_GUID>/resourceGroups/<RG>/providers/Microsoft.Web/sites/<FUNCAPP>/functions/<FUNC>`.
   This means that a span attribute MUST be used, as an Azure function app can host multiple functions that would usually share
   a TracerProvider.
-
-`cloud.platform` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
```
Comment (jsuereth) [inline docs/resource/cloud.md:43]: Why was this removed from resource?

Comment (kaiyan-sheng) [reply docs/resource/cloud.md:43]: Hmm good question! I'm looking into why `make table-generation` removed it hmm

Comment (kaiyan-sheng) [reply docs/resource/cloud.md:43]: md file for the new cloud registry is also not generated. 👀

**Comment by kaiyan-sheng** [comment]
I'm working on splitting this PR into two: one for move cloud to registry and another one to add new fields. Will close this PR.
